webapp/src/main/webapp/css/visualization/capabilitymap/key.css

```diff
@@ -0,0 +1,32 @@
+.capability, .edge, .group, .links2, .links3, .links4 {
+  text-indent: 28px;
+	font-size: 9px;
+	width: 98%;
+	background-repeat: no-repeat;
```

Comment:
The styling seems a little off:
![Screenshot from 2020-08-28 15-08-42](https://user-images.githubusercontent.com/149283/91606676-79de1e00-e940-11ea-8567-63be7987458f.png)